client/js/table-view.js

```diff
@@ -92,6 +110,22 @@ class TableView {
 		this.currentCellLocation = { col: col, row: row };
 		this.renderTableBody();
 		this.renderFormulaBar();
+		this.renderTableFooter();
+	}
+
+	sumColumnTotal(col) {
+		var total = 0;
+		// loop through each row
+		for(let row = 0; row < this.model.numRows; row++) {
+			// account for values in row 
+			if(this.model.getValue({col: col, row: row}) !== undefined) {
+				// check for words and non-number values
+				if(!isNaN(this.model.getValue({col: col, row: row}))) {
```

Comment:
This would be cleaner if you did the processing in stages:

1) gather all the values from the target column
2) filter to just numeric values
3) if you have any, sum them

Each of these stages could usefully be its own function. It would be cleaner -- each function would do one thing, and have a self-describing name. Try structuring it that way.